templates/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,24 @@
+## Description
+
+<!--- Describe your changes in detail -->
+
+Fixes: <!--- add github link -->
+
+## How Has This Been Tested?
+
+<!--- Please describe how you tested your changes. -->
+<!--- Include details of your testing environment, and the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
+
+## Screenshots (if appropriate):
+
+## Checklist:
+
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
```

Comment:
This checklist seems to be mostly for maintainers. Should we add a comment to clarify?

Reply:
Hmm... good point. I'll add a comment to it.

Reply:
@drdavella I actually removed the checklist section completely as it really isn't for the PR writer.  Does that resolve the requested change?